docs/gatsby-config.js

```diff
@@ -16,30 +16,38 @@ module.exports = {
           1: 'version-1',
         },
         sidebarCategories: {
-          null: ['index', 'getting-started', 'whats-new'],
-          Essentials: [
-            'essentials/schema',
-            'essentials/server',
-            'essentials/data',
+          null: [
+            'index',
+            'getting-started',
+            'graphql-playground',
```

Comment:
I don't think we should expose GraphQL Playground in such a prominent place on the nav, especially since it's not an Apollo property. Can we make it a subheading of another article?

Reply:
I've moved it down to "Testing" for now, if that feels all right. It might make sense for it to simply end up in the "Developer Tools" category that emerges in the core docset.